third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -173,6 +173,68 @@ func TestAccContainerCluster_withAddons(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccContainerCluster_withNotificationConfig(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+	topic := "projects/test-project/topic/test-topic"
+	newTopic := "projects/test-project/topic/test-topic-2"
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withNotificationConfig(clusterName, topic),
+			},
+			{
+				ResourceName:      "google_container_cluster.notification_config",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccContainerCluster_updateNotificationConfig(clusterName, newTopic),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
These checks aren't necessary here since the following Import steps will check these values for us.

Reply:
Removed in: https://github.com/GoogleCloudPlatform/magic-modules/pull/4012/commits/683f82a9ced16cfbe89644ca93fec425fb3d601d